frontend/app-development/features/appPublish/components/DeploymentEnvironmentLogList.test.tsx

```diff
@@ -239,50 +234,56 @@ describe('DeploymentEnvironmentLogList', () => {
     ).toBeInTheDocument();
   });
 
-  it('renders grafana link', () => {
+  it('renders log links', () => {
+    const currentDate = new Date();
+
     render({
       pipelineDeploymentList: [
         {
           ...pipelineDeployment,
           build: {
             ...pipelineDeployment.build,
+            started: currentDate.toISOString(),
+            finished: currentDate.toISOString(),
             result: BuildResult.failed,
           },
         },
       ],
     });
     expect(
-      screen.getByText(`${textMock('app_deployment.pipeline_deployment.build_result.failed')}`),
+      screen.getByText(
+        `${textMock('app_deployment.pipeline_deployment.build_result.failed.details')}`,
```

Comment:
Småpirk: Det er vel ikke nødvendig med ``${}`` her? Dette gjelder også i de andre testene.
```suggestion
        textMock('app_deployment.pipeline_deployment.build_result.failed.details'),
```